render/expected/expected.go

```diff
@@ -17,18 +17,21 @@ var (
 		ID:                unknownPseudoNode1ID,
 		LabelMajor:        "10.10.10.10",
 		Pseudo:            true,
+		NodeMetadata:      report.MakeNodeMetadata(map[string]string{}),
```

Comment:
Actually I would prefer to split

```go
func MakeNodeMetadata(m map[string]string)
```

to

```go
func MakeNodeMetadata()
func MakeNodeMetadataWith(m map[string]string)
```

to make the more-numerous empty initialisms a little nicer to read. What do you think?